aws/resource_aws_api_gateway_rest_api_test.go

```diff
@@ -144,3 +216,81 @@ resource "aws_api_gateway_rest_api" "test" {
   binary_media_types = ["application/octet-stream"]
 }
 `
+
+const testAccAWSAPIGatewayRestAPIConfigOpenAPI = `
+resource "aws_api_gateway_rest_api" "test" {
+  name = "test"
+  body = <<EOF
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "test",
+		"version": "2017-04-20T04:08:08Z"
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/test": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200 response"
+          }
+        },
+        "x-amazon-apigateway-integration": {
+          "type": "HTTP",
+          "uri": "https://www.google.de",
+          "httpMethod": "GET",
+          "responses": {
+            "default": {
+              "statusCode": 200
+            }
+          }
+        }
+      }
+    }
+  }
+}
+EOF
+}
+`
+
+const testAccAWSAPIGatewayRestAPIUpdateConfigOpenAPI = `
+resource "aws_api_gateway_rest_api" "test" {
+  name = "test"
+  body = <<EOF
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "test",
+		"version": "2017-04-20T04:08:08Z"
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/update": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200 response"
+          }
+        },
+        "x-amazon-apigateway-integration": {
+          "type": "HTTP",
+          "uri": "https://www.google.de",
+          "httpMethod": "GET",
+          "responses": {
+            "default": {
+              "statusCode": 200
+            }
+          }
+        }
+      }
+    }
+  }
+}
+EOF
+}
```

Comment:
Can you fix the space indentation here please? 😄 (mostly at the beginning of the JSON structure)